doc/cephadm/monitoring.rst

```diff
@@ -14,8 +15,27 @@ performance.  Ceph users have three options:
 #. Skip the monitoring stack completely.  Some Ceph dashboard graphs will
    not be available.
 
-Deploying monitoring with cephadm
----------------------------------
+The monitoring stack consists of `Prometheus <https://prometheus.io/>`_,
+Prometheus exporters (:ref:`mgr-prometheus`, `Node exporter
+<https://prometheus.io/docs/guides/node-exporter/>`_), `Prometheus Alert
+Manager <https://prometheus.io/docs/alerting/alertmanager/>`_ and `Grafana
+<https://grafana.com/>`_.
+
+.. note::
+
+  Please note that Prometheus' security model presumes that untrusted users
```

Comment:
```suggestion
  Prometheus' security model presumes that untrusted users

Reply:
Since it's in a called out "note" already when rendered, the beginning point is superfluous wording - but that's up to personal preference :) so feel free to ignore this one if you prefer it the other way